scripts/run-integration-tests.sh

```diff
@@ -151,10 +154,21 @@ mkdir -p "$TEST_CLUSTER_DIR"
 mkdir -p "$TEST_CONFIG_DIR"
 
 START=$SECONDS
-if [[ "$PROVISION" == true && "$RUN_KOPS_TEST" == true ]]; then
-    up-kops-cluster
-elif [[ "$PROVISION" == true ]]; then
-    up-test-cluster
+if [[ "$PROVISION" == true ]]; then
+    START=$SECONDS
+    if [[ "$RUN_BOTTLEROCKET_TEST" == true ]]; then
+        echo "TESTING downloading eksctl"
+        curl --silent --location "https://github.com/weaveworks/eksctl/releases/latest/download/eksctl_$(uname -s)_amd64.tar.gz" | tar xz -C /tmp
+        sudo mv -v /tmp/eksctl /usr/local/bin
+        eksctl create cluster --config-file ./testdata/bottlerocket.yaml
+    elif [[ "$RUN_KOPS_TEST" == true ]]; then
+        up-kops-cluster
+    else
+        up-test-cluster
+    fi
+    UP_CLUSTER_DURATION=$((SECONDS - START))
+    echo "TIMELINE: Upping test cluster took $UP_CLUSTER_DURATION seconds."
+    __cluster_created=1
```

Comment:
You don't need to set `__cluster_created=1` above, since it's set on line 173 below.